cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/service/BusinessMetadataStore.java

```diff
@@ -97,6 +99,21 @@ public Void apply(BusinessMdsIterable input) throws Exception {
   }
 
   /**
+   * @return a {@link MetadataRecord} representing all the metadata (including properties and tags) for the specified
+   * {@link Id.NamespacedId}.
+   */
+  public Set<MetadataRecord> getMetadata(final Id.NamespacedId entityId) {
```

Comment:
`BusinessMetadataStore.getMetadata(final Id.NamespacedId entityId)` still needs to return just single `MetadataRecord` object.

`DefaultMetadataAdmin..getMetadata(final Id.NamespacedId entityId)` can then make a call to metadata stores for all scopes and return a set of `MetadataRecord`, right?